mycli/main.py

```diff
@@ -111,8 +112,11 @@ def __init__(self, sqlexecute=None, prompt=None,
             self.cnf_files = [defaults_file]
 
         # Load config.
-        config_files = ([self.default_config_file] + self.system_config_files +
-                        [self.user_config_file])
+        config_files = ([self.default_config_file] + self.system_config_files)
```

Comment:
What about above this line simply doing:
```python
self.user_config_file = user_config_file or self.user_config_file
```

Then we don't have to conditionally append different files. It's kind of like what we do on line 111.